sysl2/sysl/cmd_export.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/anz-bank/sysl/sysl2/sysl/exporter"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type exportCmd struct {
+	appName string
+	out     string
+	target  string
+	format  string
+}
+
+func (p *exportCmd) Name() string            { return "export" }
+func (p *exportCmd) RequireSyslModule() bool { return true }
+
+func (p *exportCmd) Configure(app *kingpin.Application) *kingpin.CmdClause {
+	cmd := app.Command(p.Name(), "Export sysl to external types. Supported types: Swagger")
+	cmd.Flag("app-name",
+		"name of the sysl app defined in sysl model."+
+			" if there are multiple apps defined in sysl model,"+
+			" swagger will be generated only for the given app").Required().Short('a').StringVar(&p.appName)
```

Comment:
This flag isnt really required, if there are multiple apps defined and the flag is missing then flag an error, but otherwise there is little reason to force this flag on the command line.

Reply:
So if you have 100's of apps in the Sysl file, you either export none or all? You don't give the user to choose the apps he wants to export. So no its not a great idea to drop the flag altogether. What we could do is to make the option "app-name" pick up the default value of "All".

Reply:
I didnt mean drop it fully, just remove the .Required() call and default to the default app name if not provided (or error out if more than 1 app is defined but the flag isnt provided